pyfr/integrators/base.py

```diff
@@ -114,7 +135,7 @@ def get_plugin_data_prefix(name, suffix):
 
     def call_plugin_dt(self, dt):
         ta = self.tlist
-        tb = deque(np.arange(self.tcurr, self.tend, dt).tolist())
+        tb = deque(np.arange(self.tend-dt, self.tcurr, -dt).tolist()[::-1])
```

Comment:
Spacing around `-`

Reply:
Also, I think `arange` can give a reverse array directly.

Reply:
numpy arange function does not seem to have anything to reverse the array.